randovania/data/json_data/prime3.txt

```diff
@@ -552,7 +566,8 @@ Asset id: 12849086301409571765
       Trivial
 
 > Event - Metroid Hatcher; Heals? False
-  > Pickup (Energy Cell)
+  * Event Metroid Hatcher (Norion)
+  > Dock to Generator B Access
       Trivial
 
 ----------------
```

Comment:
(Below) Cargo Dock B:

- Add new node called "Room Center."
  - Connect to a new event called "Cargo Dock B Unlocked" as Trivial
  - Connect to Ship with requirement "After Cargo Dock B Unlocked and Command Visor"
  - Connect to both doors with requirement "After Cargo Dock B Unlocked"
- Conduit B should connect to Room Center as Trivial
- Generator B Access should connect to Room Center as Trivial
- Ship should connect to Room Center with requirement "After Cargo Dock B Unlocked"